README.rst

```diff
@@ -112,7 +112,7 @@ should start with a slash, but not end with one; for example::
 Python Support
 --------------
 
-Daphne requires Python 3.5 or later.
+Daphne requires Python 3.6 or later.
```

Comment:
`setup.py` still declares the Python 3.5 classifier, so remove that. It should also have `python_requires=">=3.6"`.

Reply:
This is done already, the 3.5 classifier is not there anymore. :thinking: